cockpit/gui/imageViewer/viewCanvas.py

```diff
@@ -813,3 +823,13 @@ def resetPixelScale(self):
         self.image.autoscale()
         self.histogram.lthresh, self.histogram.uthresh = self.image.getDisplayRange()
         self.Refresh()
+
+    def saveData(self, evt=None):
+        with wx.FileDialog(self, "Save image", wildcard="DV files (*.dv)|",
+                           style=wx.FD_SAVE | wx.FD_OVERWRITE_PROMPT) as fileDialog:
+            if fileDialog.ShowModal() == wx.ID_CANCEL:
+                return
+            else:
+                path = fileDialog.GetPath()
```

Comment:
I think we should be checking for `ID_OK` rather than not `ID_CANCEL` in case there's a way to exit this dialog other than cancelling. I believe some dialogs make a distinction between cancelling and closing the dialog. Checking confirmation instead of cancellation is safer.

Reply:
Checking on ID_CANCEL is straight from the wxPython docs, but good point.

Reply:
Changed to a check against ID_OK. It may not be necessary, but won't cause any problems, either. There are 3 ways I could find to close the window: the cancel button, the window's close decoration, and 'close' unde rthe window's control menu. All of these resulted in ID_CANCEL being returned.

Resolved in be12983cf8b852d8b8086146755d03aed3fa7578.